helm/qontract-reconcile/values.yaml

```diff
@@ -64,6 +64,7 @@ integrations:
       cpu: 300m
   logs:
     cloudwatch: true
+  shards: 2
```

Comment:
this may be just here temporarily, but do we intend to shard quay-mirror?  I guess we are waiting for the metrics of the other PR to adjusts the shards here.

Reply:
yes, removed